### PR TITLE
Fixed table header spaces handling

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7037c79dd05c9d135f1afd13c3c8f29ccdfde4eb:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/web_ui/__init__.py [hunk 1]
```diff
@@ -637,7 +637,7 @@ def __getattr__(self, name):
             """
             Returns Row element by header name
             """
-            return self.columns[self.table.header_indexes[name.lower()]]
+            return self.columns[self.table.header_indexes[_convert_header(name)]]
```
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: So this was an error from the beginning?

Comment (psav) [inline cfme/web_ui/__init__.py]: I think so, or it snuck in with the cache header stuff

Comment (mfalesni) [inline cfme/web_ui/__init__.py]: Ok, no objection against it.

**Pull request merged**